dist/index.js

```diff
@@ -166151,29 +166151,26 @@ const runAction = async (space) => {
         const keyName = `ephemeral-token-${branchName}`;
         const spaceKeys = await space.getApiKeys();
         let exists = false;
-        for (let i = 0; i < spaceKeys.items.length; i++) {
-            if (spaceKeys.items[i].name == keyName) {
-                exists = true;
-                break;
-            }
-        }
+        spaceKeys.some(item => item.name === keyName);
```

Comment:
Sorry, I didn't write this as a complete drop-in. You'll have to replace `exists` with this or something to its effect.